python/tools/manage_project_schemas/manage_project_schemas.py

```diff
@@ -586,11 +639,173 @@ def load_schemas(self):
+            for object_type in project_schema['object_types']:
+                if object_type['name'].lower() == 'task':
+                    continue
+                if not object_type['name'] in object_type_cache:
+                    ft_object_type = self.ensure_object_types(object_type)
+                    object_type_cache.append(object_type['name'])
+                    if not ft_object_type:
+                        self.missing_object_types.append(object_type['name'])
+            for object_type_schemas in project_schema['object_type_schemas']:
+                """{
+                        "type": "Asset Build",
+                        "statuses": [],
+                        "types": [
+                            "Lighting"
+                        ]
+                    },"""
+                object_type_data = {'name': object_type_schemas['type']}
+                ft_object_type = self.ensure_object_types(object_type_data)
+                if not ft_object_type:
+                    self.missing_object_types.append(object_type_schemas['type'])
+                statuses = object_type_schemas['statuses']
+                if statuses:
+                    for status in statuses:
+                        if status['name'] not in status_cache:
+                            new_status = self.ensure_status(status)
+                            status_cache.append(status['name'])
+                            if not new_status:
+                                self.missing_status.append(status['name'])
+                _types = object_type_schemas['types']
+                if _types:
+                    for t in _types:
+                        if t['name'] not in _type_cache:
+                            new_type = self.ensure_types(t)
+                            _type_cache.append(t['name'])
+                            if not new_type:
+                                self.missing_types.append(t['name'])
+
+        if self.missing_status:
+            logger.info("###### Missing Status ######")
+            for s in sorted(list(set(self.missing_status))):
+                logger.info(s)
+
+        if self.missing_types:
+            logger.info("###### Missing Types #######")
+            for s in sorted(list(set(self.missing_types))):
+                logger.info(s)
+
+        if self.missing_object_types:
+            logger.info("###### Missing Object Types #######")
+            for s in sorted(list(set(self.missing_object_types))):
+                logger.info(s)
+
+        # No changes has been made yet, commit to Ftrack unless dry run
+        if not self.args.dry_run:
+            logger.info('Committing Project Schemas to Ftrack...')
+            try:
+                self.session.commit()
+            except Exception as error:
+                logger.error(error, exc_info=True)
+        else:
+            pass
+            # logger.warning('Dry run, NOT committing Project Schemas to Ftrack based on JSON {}...'.format(
```

Comment:
@nebukadhezer i'd restore this warning. seems useful to know whenever you do dry_run